src/framework/draw/painter.cpp

```diff
@@ -68,6 +68,7 @@ void Painter::init()
     State st;
     st.worldTransform = m_provider->transform();
     st.isWxF = true;
+    st.transform = st.worldTransform;
```

Comment:
I'm not 200% convinced about this, but I think it's necessary to keep everything in a consistent state. Not based on practice though, only on theory..